### PR TITLE
Updated error list for iOS. Fixed getTxnRequest tests.

### PR DESCRIPTION
**Pull request by anastasia-tarasova**

### CODE REVIEW

File: tests/utils/pool.rs [hunk 1]
```diff
@@ -0,0 +1,202 @@
+extern crate time;
```
Comment (jovfer) [inline tests/utils/pool.rs]: This file seems like garbage after merge, please re-check and clean-up.

File: libindy/build-libindy-core-ios.sh [hunk 2]
```diff
@@ -22,6 +22,7 @@ fi
 echo "Packing...\n\n"
 
 cp include/*.h $WORK_DIR
+cd ../
```
Comment (jovfer) [inline libindy/build-libindy-core-ios.sh]: Please re-check the next line: should be swapped this current?

Comment (anastasia-tarasova) [inline libindy/build-libindy-core-ios.sh]: this line is deleted

File: tests/utils/pool.rs (hunk 1 above)
Comment (anastasia-tarasova) [inline tests/utils/pool.rs]: This file is deleted.

**Pull request merged**